runtime-modules/bounty/src/benchmarking.rs

```diff
@@ -367,17 +445,164 @@ benchmarks! {
             bounty_id
         ).unwrap();
 
-        let bounty = Bounty::<T>::bounties(bounty_id);
-        assert!(matches!(
-            Bounty::<T>::get_bounty_stage(&bounty),
-            BountyStage::Withdrawal {cherry_needs_withdrawal: true, ..}
-        ));
-
-    }: withdraw_creator_funding(RawOrigin::Signed(account_id), creator.clone(), bounty_id)
+    }: withdraw_creator_cherry(RawOrigin::Signed(account_id), creator.clone(), bounty_id)
     verify {
         assert!(!Bounties::<T>::contains_key(bounty_id));
         assert_last_event::<T>(Event::<T>::BountyRemoved(bounty_id).into());
     }
+
+    announce_work_entry {
+        let cherry: BalanceOf<T> = 100.into();
+        let funding_amount: BalanceOf<T> = 100.into();
+        let (account_id, member_id) = member_funded_account::<T>("member1", 0);
+
+        let creator = BountyActor::Member(member_id);
+
+        let params = BountyCreationParameters::<T>{
+            work_period: One::one(),
+            judging_period: One::one(),
+            creator: creator.clone(),
+            cherry,
+            ..Default::default()
+        };
+
+        Bounty::<T>::create_bounty(
```

Comment:
Shouldn't the bounty have a non-zero stake to take the worst branch in `validate_entrant_stake`?

Reply:
And shouldn't it have a `Closed` `contract_type` that we should benchmark in function of number of members? Since `ensure_valid_contract_type` use `contains` which I belive is `O(log(N))` in a `BTreeSet`.

Reply:
Fixed [here](https://github.com/Joystream/joystream/pull/2218)